corehq/apps/sms/forms.py

```diff
@@ -69,6 +69,18 @@
 TIME_AFTER = "AFTER"
 TIME_BETWEEN = "BETWEEN"
 
+WELCOME_RECIPIENT_NONE = 'NONE'
+WELCOME_RECIPIENT_CASE = 'CASE'
+WELCOME_RECIPIENT_MOBILE_WORKER = 'MOBILE_WORKER'
+WELCOME_RECIPIENT_ALL = 'ALL'
+
+WELCOME_RECIPIENT_CHOICES = (
+    (WELCOME_RECIPIENT_NONE, ugettext_lazy('Nobody')),
```

Comment:
Here are some specifics about the differences: https://commcare-hq.readthedocs.org/en/latest/translations.html